### PR TITLE
reverting to v1.5.23 of swagger-core

### PR DESCRIPTION
**Pull request by mgd43b**
Problem: 
swagger-core v1.6.0 introduces transitive Jackson deps that incompatible with Finatra's Jackson deps.

Solution:
Revert back to swagger-core v1.5.23 and introduce some basic tests to catch this.

Discussion:
tests are run with `sbt hello-world-example/test`

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,14 +19,25 @@ buildInfoPackage := "com.jakehschwartz.finatra.swagger"
...
 )
 
+val examplesTestLibs = Seq(
+  "com.twitter" %% "finatra-http" % twitterReleaseVersion % "test" classifier "tests",
+  "com.twitter" %% "inject-app" % twitterReleaseVersion % "test" classifier "tests",
+  "com.twitter" %% "inject-core" % twitterReleaseVersion % "test" classifier "tests",
+  "com.twitter" %% "inject-modules" % twitterReleaseVersion % "test" classifier "tests",
+  "com.twitter" %% "inject-server" % twitterReleaseVersion % "test" classifier "tests",
+  "ch.qos.logback" % "logback-classic" % "1.2.3",
+  "org.scalatest" %% "scalatest" % "3.0.8"  % Test,
```
Comment (mgd43b) [inline build.sbt:37]: Finatra is still using 3.0.x (3.1.x isn't compatible).

File: examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  override protected def beforeEach(): Unit = {
+    server.assertStarted()
+    server.assertHealthy()
+  }
+
+  def stage: Stage = Stage.DEVELOPMENT
+  override def server: EmbeddedHttpServer
+
+  protected def makeServer(serverName: String): EmbeddedHttpServer = new EmbeddedHttpServer(
```
Comment (mgd43b) [inline examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala:21]: an EmbeddedHttpServer shouldn't be shared across test classes.

File: build.sbt (hunk 1 above)
Comment (jakehschwartz) [reply build.sbt:37]: Oh right, I remember reading that now. Could you change line 27 too?

File: examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala (hunk 2 above)
Comment (jakehschwartz) [reply examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala:21]: It wouldn't be shared if it was a val, each instance of a SampleAppBaseTest would have its own

**Comment by jakehschwartz** [comment]
Thanks for this, and for adding tests! I left a few comments, let me know what you think

File: examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala (hunk 2 above)
Comment (mgd43b) [reply examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala:21]: see https://twitter.github.io/finatra/user-guide/testing/feature_tests.html#sharing-a-server-fixture-between-many-feature-tests

Comment (jakehschwartz) [reply examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala:21]: Makes sense. Could you put a comment in the code so no one accidentally changes it in the future? After that, Ill merge this in a get a patch build going.

Comment (mgd43b) [reply examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala:21]: done, thanks!

**Comment by jakehschwartz** [comment]
@mgd43b @kimxogus Version 19.12.1 is on maven central and fixes this. Thanks again for the PR!

**Pull request merged**